### PR TITLE
SQL: NPE if both comparison with float value in WHERE and LIMIT are used

### PR DESCRIPTION
**Pull request by unrealwork**
## SQL: NPE if both comparison with float value in WHERE and LIMIT are used
- [x] Tests

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/method/sql/clause/where/SqlClauseWhereWithLimitDoubleTest.java [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+
+        Response response = executeQuery(sqlQuery);
+
+        StringTable resultTable = response.readEntity(StringTable.class);
+
+        List<List<String>> expectedRows = Collections.singletonList(
+                Collections.singletonList("1.23")
+        );
+
+
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/sql/clause/where/SqlClauseWhereWithLimitDoubleTest.java]: remove extra linebreak

File: src/test/java/com/axibase/tsd/api/method/sql/clause/where/SqlClauseWhereWithLimitDoubleTest.java [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+        Response response = executeQuery(sqlQuery);
+
+        StringTable resultTable = response.readEntity(StringTable.class);
+
+        List<List<String>> expectedRows = Arrays.asList(
+                Collections.singletonList("1.23"),
+                Collections.singletonList("0.89")
+        );
+
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/sql/clause/where/SqlClauseWhereWithLimitDoubleTest.java]: linebreak

File: src/test/java/com/axibase/tsd/api/method/sql/clause/where/SqlClauseWhereWithLimitDoubleTest.java [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/sql/clause/where/SqlClauseWhereWithLimitDoubleTest.java]: linebreak

**Comment by Sild** [comment]
review done

**Comment by rmakulov** [comment]
lgtm

**Pull request merged**